### PR TITLE
Insights updates

### PR DESCRIPTION
**Pull request by teesloane**
- [x] Create glossary page
- [x] Create test insights page
- [x] Update sidebar
    - [x] add link to insights api endpoints in v2 api
    - [x] add links to new pages
    - [x] Add new icon / section
- [x] Move sumo logic to "data partnerships" page
  - [x] confirm movement of url with team working with sumo logic.
- [x] Move server related content to new "server insights" page (if necessary)

Ongoing work visible [here](http://circleci-doc-preview.s3-website-us-east-1.amazonaws.com/insights-updates-preview/2.0/insights-glossary/?section=insights)

### CODE REVIEW

File: jekyll/_cci2/insights-glossary.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+short-title: "Metric definitions for CircleCI insights"
+version:
+- Cloud
+---
+
+## Overview
+
+This document provides definitions for all the metrics in CircleCI Insights. You
+can review metrics in the Insights dashboard as well as the Insights API.
+
```
Comment (dawita66) [inline jekyll/_cci2/insights-glossary.md:14]: Can we add links to each section for easy access?

File: jekyll/_cci2/insights-glossary.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+General metrics appear across the Insights experience and can refer to different
+entities, depending on the context. For example, the `Runs` metric may refer to a
+count of workflow executions or jobs, depending on the context.
+
+| Term                    | Definition     |
+|-------------------------|----------------|
+| Runs                    | The count of executions in a selected time window. |
+| Total Duration          | The sum of execution times for all workflows in a project. |
+| P95 Duration            | The 95th percentile of execution times for an entity in a selected time window (i.e. 95% of runs completed in the same or less amount of this time). _The 95th percentile is a standard measure used to interpret performance data. It provides a measure of max value when outlier or transient values are excluded._ |
```
Comment (dawita66) [inline jekyll/_cci2/insights-glossary.md:25]: For the p95 and p50 durations, can we add a line break between the metric defnitions and italicized sections?

File: jekyll/_cci2/insights-glossary.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+| P50 (median) Duration   | The median execution time. _Medians are a better measure of central tendency than arithmetic means because they are more resilient to_ skewness _in datasets_. |
+| Total Credits           | The sum of credits consumed during execution. |
+| Success Rate            | The percentage of runs that completed successfully, calculated by 100 x (Successful Runs / All Runs) |
```
Comment (dawita66) [inline jekyll/_cci2/insights-glossary.md:28]: Would it be possible to highlight the equation in some way? It's hard to see at the moment

Comment (dawita66) [reply jekyll/_cci2/insights-glossary.md:28]: I like how it's done for Overall Success Rate in the org-level metrics table

File: jekyll/_cci2/insights-glossary.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+{: class="table table-striped"}
+
+## Organization-level metrics
+
+Organization-level metrics allow you to analyze your organization’s performance.
+
```
Comment (dawita66) [inline jekyll/_cci2/insights-glossary.md:34]: Can we add an image here to show what these metrics will look like? There's an image in the google doc. Happy to provide a higher fidelity one if needed too

File: jekyll/_cci2/insights-glossary.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+| Term                    | Definition     |
+|-------------------------|----------------|
+| Workflow Runs | The count of all workflows executions for an organization for the selected projects in the selected time frame. |
+| Total Workflow Duration | The total execution time of all workflows runs for an organization within the selected projects and time frame. |
+| Total Credits | The total credits consumed across all selected projects in an organization. |
+| Overall Success Rate | The percentage of runs that completed successfully across all runs in the selected projects and workflows. (calculated by `100 x (All Successful Runs/ All Runs)`) |
+{: class="table table-striped"}
+
+## Workflow Metrics
+
```
Comment (dawita66) [inline jekyll/_cci2/insights-glossary.md:44]: Same here. Can we add an image of a sample workflow metric view?

File: jekyll/_cci2/insights-glossary.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+## Trends
+
+Raw metric values are often difficult to interpret on their own. Trends provide
+additional context to Insights metrics by presenting a relative benchmark
+against previous performance. For instance, on the last 7-day view, trends will
+display the change in value or delta compared to the prior 7-day window.
+
+Please note that trends are available only for 24-hour, 7-day, and 30-day time
+windows. For percentile metrics like `duration`, approximation methods are used
+to find the prior window benchmarks.
```
Comment (dawita66) [inline jekyll/_cci2/insights-glossary.md:62]: Can we add the following section below the formula? Additionally, it looks like the sentence "For percentile metrics like duration, approximation methods are used to find the prior window benchmarks." can be removed following the sentence that starts with "Please note that trends..."


"**Approximate Trends**
For percentile metrics like duration, approximation methods are used to find the prior window benchmarks. 
"

File: jekyll/_cci2/insights-glossary.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+![]({{ site.baseurl }}/assets/img/docs/insights_trend_data.png)
+<sub>Screenshot depicting how trend data might appear.</sub>
+
+- **Green**: The metric is trending in the right direction.
+- **Red**: The metric is trending in the wrong direction.
+- **Grey**: A trend in either direction is not right or wrong. 
+
+Red and Green are used when describing the `Success Rate`, `Throughput` and `MTTR` metrics. Grey arrows are used when describing `Runs`, `Duration` and `Total Credits`
+
+**Percentages**
```
Comment (dawita66) [inline jekyll/_cci2/insights-glossary.md:100]: Can we add images to show how users can expect percentages, multiples, arrows, etc to look?

**Review by dawita66 — CHANGES_REQUESTED** [review]
Looks great! I left a bunch of inline comments with suggestions. Thanks!

**Comment by teesloane** [comment]
Thanks @dawita66 ! I added the requested screenshots for workflow and org metrics, as well as making the text changes.

 Is the `insights_trend_data.png` image sufficient for offering a representation of most of the data types? I would try and avoid using a single screenshot for each trend data type if we can find one that can represent them all in a single image (perhaps you have come across one? I just used a screenshot for the docs repo's trends).

File: jekyll/_cci2/insights-glossary.md (hunk 1 above)
Comment (teesloane) [reply jekyll/_cci2/insights-glossary.md:14]: I've added a table of contents to the top of the document.

File: jekyll/_cci2/insights-partnerships.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+{:.no_toc}
+You will need to create two *source collectors* on Sumo Logic which will return an HTTPS URL. Your job data will be sent to this HTTPS URL.
+
+You will need to create one called `circleci/job-collector` and another called `circleci/workflow-collector`.
+
+To create the two source collectors:
+1. From the dashboard select the **Setup Wizard**.
+2. Select **Set Up Streaming Data**.
+3. Scroll to the bottom and select **All Other Sources**.
+4. Select **HTTPS Source**
```
Comment (thomasr8) [inline jekyll/_cci2/insights-partnerships.md:89]:
```suggestion
4. Select **HTTPS Source**.
```

**Review by thomasr8 — APPROVED** [review]

File: jekyll/_cci2/insights-glossary.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
+---
+layout: classic-docs
+title: "Insights metrics glossary"
+short-title: "Insights metrics glossary"
+short-title: "Metric definitions for CircleCI insights"
+version:
+- Cloud
+---
+
+## Overview
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:10]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 2. You can fix by inserting `{: #overview }` to the next line

File: jekyll/_cci2/insights-glossary.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+
+This document provides definitions for all the metrics in CircleCI Insights. You
+can review metrics in the Insights dashboard as well as the Insights API.
+
+* TOC
+{:toc}
+
+
+## General metrics
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:19]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 11. You can fix by inserting `{: #general-metrics }` to the next line

Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:19]: MD009/no-trailing-spaces Trailing spaces Expected: 0 or 2; Actual: 1

File: jekyll/_cci2/insights-glossary.md [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+| P50 (median) Duration | The median execution time. <br/> _Medians are a better measure of central tendency than arithmetic means because they are more resilient to_ skewness _in datasets_.                                                                                                                                                         |
+| Total Credits         | The sum of credits consumed during execution.                                                                                                                                                                                                                                                                          |
+| Success Rate          | The percentage of runs that completed successfully, calculated by `100 x (Successful Runs / All Runs)`                                                                                                                                                                                                                   |
+{: class="table table-striped"}
+
+## Organization-level metrics
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:35]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 27. You can fix by inserting `{: #organization-level-metrics }` to the next line

File: jekyll/_cci2/insights-glossary.md [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+
+| Term                    | Definition     |
+|-------------------------|----------------|
+| Workflow Runs | The count of all workflows executions for an organization for the selected projects in the selected time frame. |
+| Total Workflow Duration | The total execution time of all workflows runs for an organization within the selected projects and time frame. |
+| Total Credits | The total credits consumed across all selected projects in an organization. |
+| Overall Success Rate | The percentage of runs that completed successfully across all runs in the selected projects and workflows. (calculated by `100 x (All Successful Runs/ All Runs)`) |
+{: class="table table-striped"}
+
+## Workflow Metrics
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:49]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 41. You can fix by inserting `{: #workflow-metrics }` to the next line

File: jekyll/_cci2/insights-glossary.md [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+
+| Term                    | Definition     |
+|-------------------------|----------------|
+| Time to Recovery | Total time spent in a failed state before a successful execution. |
+| Mean Time to Recovery (MTTR)| The average time it takes to get back to a successful execution from a failed one (Calculated by `total recovery time / # of failures`). <br> _If there are no failed builds (and thus no recoveries) in the current time window, the MTTR value will be empty. Empty MTTR values will be treated as 0 when calculating Trends._ |
+| Throughput | The average (mean) number of builds of any status per day. Calculated by `Sum of builds/ # of days.` |
+{: class="table table-striped"}
+
+
+## Trends
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:61]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 53. You can fix by inserting `{: #trends }` to the next line

File: jekyll/_cci2/insights-glossary.md [hunk 14]
```diff
@@ -0,0 +1,129 @@
...
+Raw metric values are often difficult to interpret on their own. Trends provide
+additional context to Insights metrics by presenting a relative benchmark
+against previous performance. For instance, on the last 7-day view, trends will
+display the change in value or delta compared to the prior 7-day window.
+
+**Please note that trends are available only for 24-hour, 7-day, and 30-day time windows.**
+
+Trends are calculated as `100 * (current value - previous value) / prior-value`.
+
+### Approximate Trends
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:72]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 64. You can fix by inserting `{: #approximate-trends }` to the next line

File: jekyll/_cci2/insights-glossary.md [hunk 15]
```diff
@@ -0,0 +1,129 @@
...
+
+For percentile metrics like duration, approximation methods are used to find the prior window benchmarks.
+
+#### Duration
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:76]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 68. You can fix by inserting `{: #duration }` to the next line

Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:76]: MD009/no-trailing-spaces Trailing spaces Expected: 0 or 2; Actual: 1

File: jekyll/_cci2/insights-glossary.md [hunk 16]
```diff
@@ -0,0 +1,129 @@
...
+Trends are calculated as `100 * (current value - previous value) / prior-value`.
+
+### Approximate Trends
+
+For percentile metrics like duration, approximation methods are used to find the prior window benchmarks.
+
+#### Duration 
+{:.no_toc}
+
+**P95 Duration**
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:79]: MD009/no-trailing-spaces Trailing spaces Expected: 0 or 2; Actual: 1

File: jekyll/_cci2/insights-glossary.md [hunk 17]
```diff
@@ -0,0 +1,129 @@
...
+
+For P95 duration, trends data is calculated by using the worst-case performance
+of the previous time window as a benchmark.
+
+For example, to calculate the last 7-day trend information, Insights selects the
+worst/largest daily P95 duration of the previous 7 days as a benchmark. For the
+last 24-hour trends, Insights selects the worst hourly P95 of the previous 24
+hours as a benchmark.
+
+**P50 Duration**
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:89]: MD009/no-trailing-spaces Trailing spaces Expected: 0 or 2; Actual: 1

File: jekyll/_cci2/insights-glossary.md [hunk 18]
```diff
@@ -0,0 +1,129 @@
...
+hours as a benchmark.
+
+**P50 Duration** 
+
+For P50 duration, Trends compares current duration with the median performance
+of the prior time window. For example, the last 30-day trends for P50 duration
+are calculated by taking the median of the daily P50 values as a benchmark.
+
+
+### Representations of trend data
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:96]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 88. You can fix by inserting `{: #representations-of-trend-data }` to the next line

File: jekyll/_cci2/insights-glossary.md [hunk 19]
```diff
@@ -0,0 +1,129 @@
...
+
+
+This section describes how your trend data may appear across various metrics.
+
+![]({{ site.baseurl }}/assets/img/docs/insights_trend_data.png)
+
+- **Green**: The metric is trending in the right direction.
+- **Red**: The metric is trending in the wrong direction.
+- **Grey**: A trend in either direction is not right or wrong.
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-glossary.md:105]: MD009/no-trailing-spaces Trailing spaces Expected: 0 or 2; Actual: 1

File: jekyll/_cci2/insights-partnerships.md [hunk 20]
```diff
@@ -0,0 +1,111 @@
+---
+layout: classic-docs
+title: "Insights Partnerships"
+short-title: "Insights Partnerships"
+version:
+- Cloud
+---
+
+## Overview
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-partnerships.md:9]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 2. You can fix by inserting `{: #overview }` to the next line

File: jekyll/_cci2/insights-partnerships.md [hunk 21]
```diff
@@ -0,0 +1,111 @@
...
+
+{:toc}
+
+This document describes how you can connect Insights data with third party
+providers.
+
+### Sumo Logic integration
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-partnerships.md:16]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 9. You can fix by inserting `{: #sumo-logic-integration }` to the next line

File: jekyll/_cci2/insights-partnerships.md [hunk 22]
```diff
@@ -0,0 +1,111 @@
...
+
+Sumo Logic users may track and visualize analytical data across all of their
+jobs on CircleCI. To do so, use the Sumo Logic Orb and Sumo Logic app
+integration from the Sumo Logic partner integrations site.
+
+
+#### The CircleCI dashboard for Sumo Logic
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-partnerships.md:23]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 16. You can fix by inserting `{: #the-circleci-dashboard-for-sumo-logic }` to the next line

File: jekyll/_cci2/insights-partnerships.md [hunk 23]
```diff
@@ -0,0 +1,111 @@
...
+- Top 10 Slowest Failed Jobs In Seconds
+- Top 10 Slowest Successful Jobs In Seconds
+
+Install the CircleCI dashboard by using the App Catalog from the dashboard home page.
+
+![header]({{ site.baseurl }}/assets/img/docs/sumologic_app_catalog.png)
+
+This dashboard receives data through the CircleCI Sumo Logic orb which must be included in your projects to be tracked.
+
+#### The Sumo Logic orb
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-partnerships.md:45]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 38. You can fix by inserting `{: #the-sumo-logic-orb }` to the next line

File: jekyll/_cci2/insights-partnerships.md [hunk 24]
```diff
@@ -0,0 +1,111 @@
...
+
+Find the latest version of the Sumo Logic orb on the [Orb Registry](https://circleci.com/developer/orbs/orb/circleci/sumologic).
+
+##### 1. Import the Sumo Logic orb.
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-partnerships.md:49]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 42. You can fix by inserting `{: #1-import-the-sumo-logic-orb }` to the next line

File: jekyll/_cci2/insights-partnerships.md [hunk 25]
```diff
@@ -0,0 +1,111 @@
...
+{:.no_toc}
+
+Add the Sumo Logic orb to your project by including the top-level `orbs` key and import `circleci/sumologic@x.y.z` as follows, replacing `x.y.z` with the latest version number at the link above.
+
+```yaml
+orbs:
+  sumologic: circleci/sumologic@x.y.z
+```
+
+##### 2. Add _workflow-collector_ to workflow.
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-partnerships.md:59]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 52. You can fix by inserting `{: #2-add-workflow-collector-to-workflow }` to the next line

File: jekyll/_cci2/insights-partnerships.md [hunk 26]
```diff
@@ -0,0 +1,111 @@
...
+      - build
+      - test:
+          requires:
+            - build
+      - deploy:
+          requires:
+            - test
+```
+
+##### 3. Create two source collectors.
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-partnerships.md:79]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 72. You can fix by inserting `{: #3-create-two-source-collectors }` to the next line

File: jekyll/_cci2/insights-partnerships.md [hunk 27]
```diff
@@ -0,0 +1,111 @@
...
+
+To create the two source collectors:
+1. From the dashboard select the **Setup Wizard**.
+2. Select **Set Up Streaming Data**.
+3. Scroll to the bottom and select **All Other Sources**.
+4. Select **HTTPS Source**
+5. For the `Source Category` enter one of the two mentioned above.
+6. Save the resulting URL.
+
+##### 4. Add environment variables.
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-partnerships.md:93]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 86. You can fix by inserting `{: #4-add-environment-variables }` to the next line

File: jekyll/_cci2/insights-partnerships.md [hunk 28]
```diff
@@ -0,0 +1,111 @@
...
+- `WORKFLOW_HTTP_SOURCE`
+
+**[How to add an environment variable to your project.]({{ site.baseurl }}/2.0/env-vars/#setting-an-environment-variable-in-a-project)**
+
+This will link the orb with your Sumo Logic dashboard.
+
+Your Sumo Logic dashboard will now begin to populate with data as each job runs on CircleCI.
+
+
+## See also
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-partnerships.md:109]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 102. You can fix by inserting `{: #see-also }` to the next line

File: jekyll/_cci2/insights-tests.md [hunk 29]
```diff
@@ -0,0 +1,44 @@
+---
+layout: classic-docs
+title: "Test Insights"
+short-title: "Using Test Insights"
+description: "Using Test Insights"
+version:
+- Cloud
+---
+
+## Overview
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-tests.md:10]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 2. You can fix by inserting `{: #overview }` to the next line

File: jekyll/_cci2/insights-tests.md [hunk 30]
```diff
@@ -0,0 +1,44 @@
...
+
+Using CircleCI Test Insights, users are able to analyze their test performance
+within their most recent executions and identify opportunities for test
+optimization. To access Test Insights, select the **Tests** tab on the
+**Workflow Insights** page.
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-tests.md:15]: MD009/no-trailing-spaces Trailing spaces Expected: 0 or 2; Actual: 1

File: jekyll/_cci2/insights-tests.md [hunk 31]
```diff
@@ -0,0 +1,44 @@
...
+## Overview
+
+Using CircleCI Test Insights, users are able to analyze their test performance
+within their most recent executions and identify opportunities for test
+optimization. To access Test Insights, select the **Tests** tab on the
+**Workflow Insights** page. 
+
+The following test insights are available:
+
+### Performance Summary
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-tests.md:19]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 11. You can fix by inserting `{: #performance-summary }` to the next line

File: jekyll/_cci2/insights-tests.md [hunk 32]
```diff
@@ -0,0 +1,44 @@
...
+
+
+![]({{ site.baseurl }}/assets/img/docs/insights-test-performance.png)
+
+The summary chart displays test performance across the most recent 10 workflow
+runs. Users can hover over each bar to review the test count, skipped tests, and
+success rate of their tests.
+
+### Top 10 Most Failed Tests
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-tests.md:28]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 20. You can fix by inserting `{: #top-10-most-failed-tests }` to the next line

Comment (circleci-docs) [inline jekyll/_cci2/insights-tests.md:28]: MD009/no-trailing-spaces Trailing spaces Expected: 0 or 2; Actual: 1

File: jekyll/_cci2/insights-tests.md [hunk 33]
```diff
@@ -0,0 +1,44 @@
...
+### Top 10 Most Failed Tests 
+
+![]({{ site.baseurl }}/assets/img/docs/insights-test-most-failed.png)
+
+Users can view the 10 tests with the lowest success rates in their most recent
+pipeline executions. The table provides the Test name, the associated Job, run
+time, and success rate.
+
+
+### Top 10 Slowest Tests
```
Comment (circleci-docs) [inline jekyll/_cci2/insights-tests.md:37]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 29. You can fix by inserting `{: #top-10-slowest-tests }` to the next line

Comment (circleci-docs) [inline jekyll/_cci2/insights-tests.md:37]: MD009/no-trailing-spaces Trailing spaces Expected: 0 or 2; Actual: 1

**Pull request merged**